operators/derive/to_dp.py

```diff
@@ -7,6 +9,21 @@
 from . import helpers
 
 
+def merge_array_fields(fieldnames):
+    def func(r):
+        # get rid of null fields
+        vals = filter(None, [r[name] for name in fieldnames])
```

Comment:
I didn't know that worked!

Reply:
Yes, simple trick to remove falsy values